File: SofiForce.Models/Models/EntityModels/ManufacturerModel.cs
using System;

namespace Models
{
    public class ManufacturerModel
    {
        public int ManufacturerId { get; set; }
        public string ManufacturerCode { get; set; }
        public string ManufacturerNameEn { get; set; }
        public string ManufacturerNameAr { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public bool? IsActive { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ManufacturerModel.ts
import { CarModel } from "./carModel";

export interface ManufacturerModel {
    manufacturerId: number;
    manufacturerCode: string;
    manufacturerNameEn: string;
    manufacturerNameAr: string;
    color: string;
    icon: string;
    displayOrder: number | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    isActive: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}